frontend/components/feedback-task/container/fields/text-field/TextField.component.vue

```diff
@@ -48,6 +48,24 @@ export default {
   setup(props) {
     return useTextFieldViewModel(props);
   },
+  computed: {
+    renderedText() {
```

Comment:
From my point of view, the TextField does not have the responsibility to manage this case.
I think the problem is in the Markdown, right? did you have time to reade https://marked.js.org/using_pro ??